ext/drreg/drreg.h

```diff
@@ -345,6 +357,17 @@ drreg_status_t
 drreg_reserve_register(void *drcontext, instrlist_t *ilist, instr_t *where,
                        drvector_t *reg_allowed, OUT reg_id_t *reg);
 
+DR_EXPORT
+/**
+ * Similar to \p drreg_reserve_register, but for a particular spill class.
```

Comment:
I would s/Similar to/Identical to/ or otherwise replace the vague similar with a more precise description.

Reply:
Done.